### ISSUE TITLE
Occurs some loss = nan steps, is it rational? 

### ISSUE THREAD

**Issue by OceannTwT** [hiyouga/ChatGLM-Efficient-Tuning#49]
I am using your example to train ppo, while the logging show some loss = nan, I am curious about that if it is rational?

some plots:

{'loss': 0.3290, 'reward': -2.0304, 'learning_rate': 5e-05}
  0%|                                                                                                                                                                                                                                                                     | 1/13000 [00:10<36:34:36, 10.13s/it]{'loss': nan, 'reward': 5.7646, 'learning_rate': 5e-05}
  0%|                                                                                                                                                                                                                                                                     | 2/13000 [00:28<54:33:11, 15.11s/it{'loss': 0.2527, 'reward': 1.2237, 'learning_rate': 5e-05}
  0%|                                                                                                                                                                                                                                                                     | 3/13000 [00:39<47:01:51, 13.03s/it]{'loss': 0.1512, 'reward': 10.7681, 'learning_rate': 5e-05}
  0%|                                                                                                                                                                                                                                                                     | 4/13000 [01:01<60:36:15, 16.79s/it]{'loss': 0.0769, 'reward': 7.0280, 'learning_rate': 5e-05}
  0%|                                                                                                                                                                                                                                                                     | 5/13000 [01:20<63:06:18, 17.48s/it]{'loss': 0.1685, 'reward': 12.2049, 'learning_rate': 5e-05}
  0%|                                                                                                                                                                                                                                                                     | 6/13000 [01:28<51:51:58, 14.37s/it]

**Comment by hiyouga** [comment]
Please update the code using `git pull`.
